From b0e226cf561d9b646c737a64586be93772e8242a Mon Sep 17 00:00:00 2001
From: Bin Meng <bmeng.cn@gmail.com>
Date: Wed, 6 Apr 2022 22:04:04 +0800
Subject: [PATCH 5/6] makefile: Avoid absolute path in CXX for mingw* builds

At present CXX specifies absolute path for the MinGW GCC compiler.
But it only applies to MinGW cross compiler on the Linux host.
On a Windows host with MSYS2, MinGW GCC is normally installed at
/mingw{32,64}/bin which does not work with current makefile.

Remove the absolute path prefix, as on both Linux and Windows hosts
MinGW GCC compiler path is already in the PATH environment.

Signed-off-by: Bin Meng <bmeng.cn@gmail.com>
---
 Makefile.mingw64 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.mingw64 b/Makefile.mingw64
index 14b4b8c..55fd5ff 100644
--- a/Makefile.mingw64
+++ b/Makefile.mingw64
@@ -1,6 +1,6 @@
 CC=/usr/bin/x86_64-w64-mingw32-gcc
-CXX=/usr/bin/x86_64-w64-mingw32-g++
-STRIP=/usr/bin/x86_64-w64-mingw32-strip
+CXX=x86_64-w64-mingw32-g++
+STRIP=x86_64-w64-mingw32-strip
 CFLAGS=-O2 -Wall -static -static-libgcc -static-libstdc++  -D_FILE_OFFSET_BITS=64 -g
 CXXFLAGS=-O2 -Wall -std=c++14 -D_FILE_OFFSET_BITS=64
 #CXXFLAGS=-O2 -Wall -D_FILE_OFFSET_BITS=64 -I /usr/local/include -I/opt/local/include -g
-- 
2.35.1

